data/sheragi/sheragi outfits.txt

```diff
@@ -225,7 +225,7 @@ outfit "Sheragi Anti-Missile"
 		"reload" 60
 		"firing energy" 220
 		"firing heat" 440
-	description "This Sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but it comes at the expense of a long recharge time."
+	description "This Sheragi Anti-Missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but it comes at the expense of a long recharge time."
```

Comment:
I think this is probably the most prominent example of my stance on things. The intended context of "This anti-missile system created by the Sheragi" is pretty clear, but, because "Sheragi Anti-Missile" is the proper thing itself, the actual, proper context -becomes- "This Sheragi Anti-Missile system" as corrected to here.

Reply:
I wouldn't have changed it; this could just be one example of a Sheragi anti-missile system. We don't know, so saying that it is THE Sheragi Anti-Missile might be okay, but I prefer it without capitalisation.

Reply:
Even if the Sheragi do have other anti-missile systems, and even if we do eventually get access to them, until/unless this name is changed, it will always be THE "Sheragi Anti-Missile," because that is, unfortunately, literally what it is.

Reply:
Then I think it should be like this:
```suggestion
	description "The Sheragi Anti-Missile fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but it comes at the expense of a long recharge time."
```